### PR TITLE
feature: erigon snapshot support

### PR DESCRIPTION
**Pull request by rebelArtists**
**purpose:**
- add erigon mumbai archive incremental snapshot support to download/extract script
- make checksum verification optional for users to speed up downloads
- add erigon mumbai archive recommend disk size guidance
- clarify erigon mainnet archive temporarily under maintenance per Polygon eng team

**tests:**
- e2e download/extract/erigon client start fully tested successfully on prod nodes, and snapshot download page also updated accordingly:
image

### CODE REVIEW

File: docs/pos/reference/snapshot-instructions-heimdall-bor.md [hunk 1]
```diff
@@ -19,20 +19,20 @@ image: https://wiki.polygon.technology/img/polygon-wiki.png
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-When setting up a new sentry, validator, or full node server, it is recommended that you use snapshots for faster syncing without having to sync over the network. Using snapshots will save you several days for both Heimdall and Bor.
+When setting up a new sentry, validator, or full node server, it is recommended that you use snapshots for faster syncing without having to sync over the network. Using snapshots will save you several days for both Heimdall and Bor. Note: We no longer support bor archive snapshots due to unsustainable data growth.
```
Comment (DannyS03) [inline docs/pos/reference/snapshot-instructions-heimdall-bor.md:22]:
```suggestion
When setting up a new sentry, validator, or full node server, it is recommended that you use snapshots for faster syncing without having to sync over the network. Using snapshots will save you several days for both Heimdall and Bor. **Note: We no longer support bor archive snapshots due to unsustainable data growth.** 
```

File: docs/pos/reference/snapshot-instructions-heimdall-bor.md [hunk 2]
```diff
@@ -19,20 +19,20 @@ image: https://wiki.polygon.technology/img/polygon-wiki.png
...
+When setting up a new sentry, validator, or full node server, it is recommended that you use snapshots for faster syncing without having to sync over the network. Using snapshots will save you several days for both Heimdall and Bor. Note: We no longer support bor archive snapshots due to unsustainable data growth. 
 
 :::tip
 
 For the latest snapshot, please visit [<ins>Polygon Chains Snapshots</ins>](https://snapshot.polygon.technology/).
 
 :::
 
-## Heimdall/Bor Snapshots
+## Heimdall/Bor/Erigon Snapshots
```
Comment (DannyS03) [inline docs/pos/reference/snapshot-instructions-heimdall-bor.md:30]:
```suggestion
## Client Snapshots
```

File: docs/pos/reference/snapshot-instructions-heimdall-bor.md [hunk 3]
```diff
@@ -108,7 +129,7 @@ extract_files $client-$network-incremental-compiled-files.txt
 aria2c -c -m 0 -x6 -s6 -i heimdall-$network-incremental-compiled-files.txt --max-concurrent-downloads=1
 ```
 
-Once the extraction is complete, ensure that you update the datadir configuration of your Heimdall or Bor client to point to the path where the extracted data is located.
+Once the extraction is complete, ensure that you update the datadir configuration of your Heimdall/Bor/Erigon client to point to the path where the extracted data is located.
```
Comment (DannyS03) [inline docs/pos/reference/snapshot-instructions-heimdall-bor.md:132]:
```suggestion
Once the extraction is complete, ensure that you update the datadir configuration of your client to point to the path where the extracted data is located.
```

**Review by DannyS03 — APPROVED** [review]
thanks @rebelArtists, looks good - just some minor suggestions on writing style.

**Comment by rebelArtists** [comment]
> thanks @rebelArtists, looks good - just some minor suggestions on writing style.

thanks @DannyS03 suggestions look good, accepted

**Pull request merged**